index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <link rel="stylesheet" href="style.css">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <div class="wrapper">
+    <div class="bar-nav">
+        <h1>My Logo</h1>
+        <img src="image/Union.png" alt="Menu icon">
+    </div>
+    <div class="container">
+      <section class="detail one">
+        I'm Mariane<br>Glad to see<br>you
+      </section>
+      <section class="detail two">
+        I am a software developer! I can help you build a product, feature or website. Look through some of my work and experience. If you like what you see and have a project you need code, don't hesitate to contact me
+      </section>
+      <section class="detail link">
+        <a href="#">LET'S CONNECT</a>
+      </section>
+      <section class="detail nest">
+          <img src="image/Vector-(4).png" alt="Email icon" class="space">
+          <img src="image/Linkedin-icon.png" alt="Social media" class="space">
+          <img src="image/Vector-(1).png" alt="Email icon" class="space">
+          <img src="image/Vector-(2).png" alt="Social media" class="space">
+          <img src="image/Vector-(3).png" alt="social media" class="space">
+      </section>
```

Comment:
- [ ]  As this is a list of elements, I suggest you use an unordered list `<ul>` tag, with each element being a list item that includes a link to your social media accounts.

Reply:
done